node_package/src/RSCServerRoot.ts

```
@@ -0,0 +1,111 @@
+      'is set to true.'
+    );
+  }
+
+  const ssrManifest = await createSSRManifest(
+    railsContext.reactServerClientManifestFileName,
+    railsContext.reactClientManifestFileName
+  );
+  const rscPayloadStream = await generateRSCPayload(
+    componentName,
+    componentProps,
+    railsContext.serverSideRSCPayloadParameters
```

ESLint

[error] Insert `,`

(prettier/prettier)